src/com/rcv/Tabulator.java

```diff
@@ -254,14 +255,43 @@ private BigDecimal getThreshold(Map<String, BigDecimal> currentRoundCandidateToT
     return threshold;
   }
 
+  // purpose: determine if we should continue tabulating based on how many winners and losers
+  // have been selected and the configuration settings in use
+  // return: true if we should continue tabulating
+  private boolean shouldContinueTabulating() {
+    // how many candidates have already been eliminated
+    int eliminatedCandidates = candidateToRoundEliminated.keySet().size();
+//    // how many winners have been selected
+    int winners = winnerToRound.size();
+    if(config.continueTabulationTillAllCandidatesEliminated()) {
+      return (eliminatedCandidates + winners) < config.numCandidates();
+    } else {
+      return winners < config.numberOfWinners();
+    }
+//    return true;
+  }
+  
+  // function: isContinuing
+  // purpose: returns true if candidate is continuing with respect to tabulation
+  // this handles continued tabulation after a winner has been chosen for the
+  // continueTabulationTillAllCandidatesEliminated setting
+  // returns: true if we should continue tabulating
+  private Boolean isContinuing(String candidate) {
```

Comment:
I would call this `isCandidateContinuing` to clarify that it's not answering this question about the Tabulator object itself.